tests/test_Component_createStats.py

```diff
@@ -0,0 +1,47 @@
+# Automatically generated SST Python input
+import sst
+
+# Define SST core options
+sst.setProgramOption("timebase", "1 ps")
```

Comment:
Remove this line. In general, we are trying to discourage people from setting timebase.  We made the mistake of adding it into early examples.